### ISSUE TITLE
Are globs and wildcards supported?

### ISSUE THREAD

**Issue by nbsp** [djnicholson/release-action#6]
A file I want to add is called `my-app-${version}.exe`, and I don't want to update the action file each time I change the version. I'd like to use some sort of wildcard like `*`, but it doesn't seem to be supported.

**Comment by djnicholson** [comment]
You're correct, they are not supported, but easily could be.  I'd accept a PR if you want to make the change.  Otherwise I'll add it next time I have some spare time to work in this repo.
